forms-flow-api/src/formsflow_api/app.py

```diff
@@ -49,7 +52,13 @@ def create_app(run_mode=os.getenv("FLASK_ENV", "production")):
     logs = logging.StreamHandler()
 
     logs.setFormatter(CustomFormatter())
-    app.logger.handlers = [logs]
+    if not os.path.exists("logs"):
```

Comment:
You could move this to utils package and expose as a def, so that every application just call something like;
`register_log_handlers(app)`

Reply:
Also please look into adding a logger format with timestamp